website/content/docs/upgrading/index.mdx

```diff
@@ -96,9 +75,13 @@ client, and entrypoints in any order for a standard, backwards compatible upgrad
 1. Check any upgrade notes for the version you're upgrading to and verify
    there are no issues that will affect your upgrade.
 
-2. Shut down the previous server version A.
+2. [Take a snapshot of the server](https://www.waypointproject.io/commands/server-snapshot)
+
+3. Shut down the previous server version A.
+
+4. Start the new server version B.
 
-3. Start the new server version B.
+5. [Restore the server snapshot](https://www.waypointproject.io/commands/server-restore)
```

Comment:
nit: should we add a next step like "verify the connection with `waypoint context verify` or something?

Reply:
Sorry, I smashed the merge button before I saw this 😔 https://github.com/hashicorp/waypoint/pull/2236